acceptance/setup/git/pre-suite/030_PuppetMasterSanity.rb

```diff
@@ -1,6 +1,6 @@
 test_name "Puppet Master sanity checks: PID file and SSL dir creation"
 
-pidfile = '/var/lib/puppet/run/master.pid'
+pidfile = '/var/run/puppetlabs/agent/master.pid'
```

Comment:
Fair point - the naming is confusing. Glad we won't be living with this in puppet 5 :)

https://confluence.puppetlabs.com/display/DEL/Unified+Puppet+Agent+Layout only specifies a single pidir, but if the naming will be too confusing perhaps we should either: a) have 2 directories, 1 for master, 1 for agent? or b) have one but at /var/run/puppetlabs?

/cc @haus